package.json

```diff
@@ -32,27 +32,22 @@
   },
   "homepage": "https://graphql-extended.github.io",
   "dependencies": {
-    "apollo-server-express": "^1.3.6",
-    "apollo-upload-server": "^7.1.0",
+    "apollo-server-express": "^2.15.1",
     "gqlx-js": "^0.2.9",
     "graphql": "^14.4.1",
     "graphql-subscriptions": "^1.1.0",
-    "graphql-tools": "^4.0.5",
     "graphql-type-json": "^0.2.4",
     "subscriptions-transport-ws": "^0.9.16"
   },
   "peerDependencies": {
-    "express": "3.x || 4.x"
+    "express": "^4.17.1"
```

Comment:
I would just remove this. Its a transitive dependency now.

Formerly this was a "real" middleware; now its more or less an Express replacement that just uses Express underneath.

Reply:
removed as discussed